sql/core/src/main/scala/org/apache/spark/sql/execution/python/TransformWithStateInPandasExec.scala

```diff
@@ -154,6 +178,7 @@ case class TransformWithStateInPandasExec(
           // by the upstream (consumer) operators in addition to the processing in this operator.
           allUpdatesTimeMs += NANOSECONDS.toMillis(System.nanoTime - updatesStartTimeNs)
           commitTimeMs += timeTakenMs {
+            processorHandle.doTtlCleanup()
```

Comment:
Is this a bugfix for existing bug? If then please file a new JIRA ticket and submit a new PR. Let's not mix up with different things.